src/zc/buildout/buildout.txt

```diff
@@ -1264,10 +1264,10 @@ Expressions must not contain either the ``#`` or the ``;`` character.
 Adding and removing options
 ---------------------------
 
-We can append and remove values to an option by using the + and -
+We can append and remove values to an option by using the ``+`` and ``-``
 operators.
 
-This is illustrated below; first we define a base configuration.
+This is illustrated below; first we define a base configuration::
```

Comment:
Well, Vim's highlighting reveals the issue, but the fact is that:
> A text block that is indented relative to the preceding text, without preceding markup indicating it to be a literal block or other content, is a block quote. [Block Quotes](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#block-quotes)

So I do think marking code blocks as correct RST (literal blocks or using [directives](http://sphinx-doc.org/markup/index.html#sphinxmarkup)) has value.